.github/workflows/ci.yml

```diff
@@ -5,143 +5,314 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
+env:
+  DEFAULT_NODE_VERSION: 22.x
+  DOCUMENTATION_BRANCH: feat/simplify-extension-ignores
+
 jobs:
 
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Ensure line endings are consistent
+        run: git config --global core.autocrlf input
+      - name: Use Node.js ${{ env.DEFAULT_NODE_VERSION }}
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ env.DEFAULT_NODE_VERSION }}
+      - name: Enable Corepack
```

Comment:
Let's not use corepack, as Node will remove it: https://thenewstack.io/node-removes-corepack-bun-runs-native-c-from-javascript/

Reply:
Oh, that is sad news. I think bumping the Yarn version without Corepack available would be really annoying for anyone working on the repo in that case, because manually setting up the correct version for everything would be confusing. :frowning: